std/src/main/scala/cats/std/future.scala

```diff
@@ -0,0 +1,54 @@
+package cats
+package std
+
+import scala.concurrent.{Await, ExecutionContext, Future}
+import scala.concurrent.duration.FiniteDuration
+
+trait FutureInstances extends FutureInstances1 {
+
+  implicit def futureInstance(implicit ec: ExecutionContext): Monad[Future] with CoflatMap[Future] =
+    new FutureCoflatMap with Monad[Future]{
+
+      def pure[A](x: A): Future[A] = Future.successful(x)
+
+      def flatMap[A, B](fa: Future[A])(f: A => Future[B]): Future[B] = fa.flatMap(f)
+
+      override def map[A, B](fa: Future[A])(f: A => B): Future[B] = fa.map(f)
+    }
+
+  implicit def futureSemigroup[A](implicit A: Semigroup[A], ec: ExecutionContext): Semigroup[Future[A]] =
+    new FutureSemigroup[A]() {}
```

Comment:
I think you probably want to remove the `{}` at the end. Currently I think it's creating a new anonymous subclass of `FuureSemigroup` instead of just creating a new instance.